Task6_GettingStartedwithAWS/Models/AwsCredentials.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
```

Comment:
Remove unused imports. Check this in all of the files

Reply:
unused imports removed in all files.